contracts/registrar/ZNSSubRegistrar.sol

```diff
@@ -99,14 +105,30 @@ contract ZNSSubRegistrar is AAccessControlled, ARegistryWired, UUPSUpgradeable,
             !registry.exists(domainHash),
             "ZNSSubRegistrar: Subdomain already exists"
         );
+        require(
```

Comment:
I was mistaken with where the `invalid token id` error was coming from. Initially, this was a short circuit because if the tx fails much deeper in the tx when trying to mint it would be more costly and the intent was to try to be less costly if a user makes a mistake and the tx reverts, but it isn't needed